src/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -0,0 +1,478 @@
+
+                // RAINBOW (outside the BMP and does not case)
+                yield return new object[] { cultureName, "\U0001F308", "\U0001F308" };
+
+                // Unicode defines some codepoints which expand into multiple codepoints
+                // when cased (see SpecialCasing.txt from UNIDATA for some examples). We have never done
+                // these sorts of expansions, since it would cause string lengths to change when cased,
+                // which is non-intuitive. In addition, there are some context sensitive mappings which
+                // we also don't preform.
+                // Greek Capital Letter Sigma (does not to case to U+03C2 with "final sigma" rule).
+                yield return new object[] { cultureName, "\u03A3", "\u03C3" };
+            }
+
+            foreach (string cultureName in new string[] { "tr", "tr-TR", "az", "az-Latn-AZ" })
+            {
+                yield return new object[] { cultureName, "\u0130", "i" };
+                yield return new object[] { cultureName, "i", "i" };
+                yield return new object[] { cultureName, "I", "\u0131" };
+                yield return new object[] { cultureName, "HI!", "h\u0131!" };
+                yield return new object[] { cultureName, "HI\n\0H\u0130\t!", "h\u0131\n\0hi\u0009!" };
+            }
+
+            // ICU has special tailoring for the en-US-POSIX locale which treats "i" and "I" as different letters
+            // instead of two letters with a case difference during collation.  Make sure this doesn't confuse our
+            // casing implementation, which uses collation to understand if we need to do Turkish casing or not.
+            if (!PlatformDetection.IsWindows)
+            {
+                yield return new object[] { "en-US-POSIX", "I", "i" };
+            }
+        }
+
+        public void TestToLower(string name, string str, string expected)
+        {
+            Assert.Equal(expected, new CultureInfo(name).TextInfo.ToLower(str));
+            if (str.Length == 1)
+            {
+                Assert.Equal(expected[0], new CultureInfo(name).TextInfo.ToLower(str[0]));
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(ToLower_TestData))]
+        public void ToLower(string name, string str, string expected)
+        {
+            TestToLower(name, str, expected);
+        }
+
+        [Theory]
+        [MemberData(nameof(ToLower_TestData_netcore))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void ToLower_Netcore(string name, string str, string expected)
+        {
+            TestToLower(name, str, expected);
+        }
+
+        [Theory]
+        [MemberData(nameof(ToLower_TestData_net46))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.Uap)]
+        public void ToLower_net46(string name, string str, string expected)
```

Comment:
I believe we decided to stop adding netfx-specific tests.
cc: @danmosemsft

Reply:
Correct, we want to remove those where an assembly is not shipped for .NET Framework. This assembly is not.
I should have tagged you @hughbe on the discussion/reasoning here: https://github.com/dotnet/corefx/issues/35088